web/src/components/graph/SystemGraph.tsx

```diff
@@ -111,7 +107,7 @@ export function ThresholdBarGraph({
         tickPlacement: "on",
         labels: {
           rotate: 0,
-          offsetX: -18,
+          offsetX: isMobileOnly ? -18 : 0,
```

Comment:
should this be applicable to tablets too?

Reply:
this is actually copied from the other line which works this way. The reason being that a tablet has more space so the offset is not needed